packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1338,7 +1344,7 @@ export default class SalesforceCRMService implements CRM {
     }
 
     const leadsQuery = await conn.query(
```

Comment:
I don't get this entirely, there's prioritisation of contacts over leads but if there are contacts AND leads the personRecord is overwritten with the lead record, right?

Reply:
@keithwillcode @joeauyeung is making a follow-up PR for this; will approve this change.